src/controllers/notifications.js

```diff
@@ -1,67 +0,0 @@
-'use strict'
```

Comment:
Yes, we're talking about a client with some database-backed state (that's why I called it _pseudo-robust).